<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreatingEvents</class>
 <widget class="QDialog" name="CreatingEvents">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #E6E6E6;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="10" column="1">
    <widget class="QPushButton" name="CancelButton">
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border-radius: 10px;
	background-color: #12CDD4;
	font-size: 18px;
border: none;
    text-align: center;
    text-decoration: none;
    font-size: 16px;
    margin: 4px 2px;
}
QPushButton:hover { 
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #12CDD4, stop: 1 #a1e1e3 );
	border: 2px solid black;
}
QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #a1e1e3, stop: 1 #12CDD4);
	border: 2px solid black;
}</string>
     </property>
     <property name="text">
      <string>Отмена</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Описание</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="LocationLineEdit">
     <property name="styleSheet">
      <string notr="true">background-color: white;</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Тип</string>
     </property>
    </widget>
   </item>
   <item row="10" column="3">
    <widget class="QPushButton" name="SaveButton">
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border-radius: 10px;
	background-color: #12CDD4;
	font-size: 18px;
border: none;
    text-align: center;
    text-decoration: none;
    font-size: 16px;
    margin: 4px 2px;
}
QPushButton:hover { 
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #12CDD4, stop: 1 #a1e1e3 );
	border: 2px solid black;
}
QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #a1e1e3, stop: 1 #12CDD4);
	border: 2px solid black;
}</string>
     </property>
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Выбрать информируемые лица</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="TypeLineEdit">
     <property name="styleSheet">
      <string notr="true">background-color: white;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Место провидения</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Название</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QTextEdit" name="DescriptionTextEdit">
     <property name="styleSheet">
      <string notr="true">background-color: white;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Выбрать участников события</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="TitleLineEdit">
     <property name="styleSheet">
      <string notr="true">background-color: white;</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Дата начала</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Окончание</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QTimeEdit" name="EndTimeEdit">
     <property name="styleSheet">
      <string notr="true">background-color: white;</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QTimeEdit" name="StartTimeEdit">
     <property name="styleSheet">
      <string notr="true">background-color: white;</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QDateEdit" name="StartDateEdit">
     <property name="styleSheet">
      <string notr="true">background-color: white;</string>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>12</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2024</year>
       <month>4</month>
       <day>30</day>
      </datetime>
     </property>
    </widget>
   </item>
   <item row="1" column="4" rowspan="5">
    <widget class="QListWidget" name="InformedPersonslistWidget">
     <property name="styleSheet">
      <string notr="true">background-color: white;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" rowspan="5">
    <widget class="QListWidget" name="ParticipantslistWidget">
     <property name="styleSheet">
      <string notr="true">background-color: white;</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Приоритет</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="PrioritycomboBox">
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QLabel" name="label_10">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Периодичность события</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>Без повторений</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Каждый день</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Раз в неделю</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Раз в месяц</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Раз в год</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>TitleLineEdit</tabstop>
  <tabstop>DescriptionTextEdit</tabstop>
  <tabstop>ParticipantslistWidget</tabstop>
  <tabstop>InformedPersonslistWidget</tabstop>
  <tabstop>LocationLineEdit</tabstop>
  <tabstop>TypeLineEdit</tabstop>
  <tabstop>PrioritycomboBox</tabstop>
  <tabstop>StartDateEdit</tabstop>
  <tabstop>StartTimeEdit</tabstop>
  <tabstop>EndTimeEdit</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>CancelButton</tabstop>
  <tabstop>SaveButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
